apps/frontend/src/app/3_organisms/Header/components/BridgeMenuItem/BridgeMenuItem.tsx

```diff
@@ -88,78 +90,94 @@ export const BridgeMenuItem: FC<BridgeMenuItemProps> = ({ dataAttribute }) => {
-          })}
-          onClick={handleBtcClicked}
-        />
-        <MenuItem
-          key={t('header.nav.bridges.subMenu.runeBridge')}
-          text={t('header.nav.bridges.subMenu.runeBridge')}
-          label={
-            !isMobile && t('header.nav.bridges.subMenu.runeBridgeDescription')
-          }
-          dataAttribute={`dapp-menu-runeBridge`}
-          className="no-underline"
-          onClick={handleRunesClick}
-        />
-        <MenuItem
-          key={t('header.nav.bridges.subMenu.tokenBridge')}
-          text={t('header.nav.bridges.subMenu.tokenBridge')}
-          label={
-            !isMobile && t('header.nav.bridges.subMenu.tokenBridgeDescription')
-          }
-          dataAttribute={`dapp-menu-tokenBridge`}
-          className={classNames('no-underline', {
-            hidden: !isRskChain(chainId),
-          })}
-          onClick={handleStablecoinsClick}
-        />
-        <MenuItem
-          key={t('header.nav.bridges.subMenu.ercBridge')}
-          text={t('header.nav.bridges.subMenu.ercBridge')}
-          label={
-            !isMobile && t('header.nav.bridges.subMenu.ercBridgeDescription')
-          }
-          dataAttribute={`dapp-menu-ercBridge`}
-          className={classNames('no-underline', {
-            hidden: isRskChain(chainId),
-          })}
-          onClick={handleErcClick}
-        />
-      </Menu>
-    </NavDropdown>
+      <div>
+        <NavDropdown
+          text={t('header.nav.bridges.title')}
+          className={classNames(
+            'text-gray-30 font-normal text-sm min-w-auto w-full lg:w-auto rounded bg-gray-70 border-gray-50',
+            {
+              'opacity-30': !account,
+              'hover:bg-gray-70 hover:text-gray-10': !!account,
+            },
+          )}
+          disabled={!account}
+          closeOnClick={!isMobile}
+          dataAttribute={dataAttribute}
+        >
+          <Menu className="rounded-t-none rounded-b px-2 py-3 lg:rounded lg:p-1">
+            {isRskChain(chainId) && (
+              <MenuItem
+                key={t('header.nav.bridges.subMenu.ethBridge')}
+                text={t('header.nav.bridges.subMenu.ethBridge')}
+                label={
+                  !isMobile &&
+                  t('header.nav.bridges.subMenu.ethBridgeDescription')
+                }
+                dataAttribute={`dapp-menu-ethBridge`}
+                className={classNames('no-underline', {
+                  hidden: isRskChain(chainId),
+                })}
+                onClick={handleEthClicked}
+              />
+            )}
+            <MenuItem
+              key={t('header.nav.bridges.subMenu.btcBridge')}
+              text={t('header.nav.bridges.subMenu.btcBridge')}
+              label={
+                !isMobile && isRskChain(chainId)
+                  ? t('header.nav.bridges.subMenu.btcBridgeDescription')
+                  : t('header.nav.bridges.subMenu.ethBridgeDescription')
+              }
```

Comment:
```suggestion
                !isMobile && 
                   t('header.nav.bridges.subMenu.btcBridgeDescription')
              }
```
This is hidden if you are not on `RSK` so that condition doesn't make sense. Also, it's the only label allowed on mobiles so I think we should make it consistent.